src/Package.php

```diff
@@ -75,6 +64,17 @@ public static function init() {
 			return;
 		}
 
+		$feature_plugin_instance = FeaturePlugin::instance();
+		$satisfied_dependencies  = $feature_plugin_instance->has_satisfied_dependencies();
```

Comment:
```suggestion
		$satisfied_dependencies  = is_callable( array( $feature_plugin_instance, 'has_satisfied_dependencies' ) ) && $feature_plugin_instance->has_satisfied_dependencies();
```

Reply:
With that change copied to the WC package folder, this passed unit tests running against the `master` branch (1.0.0).